.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+name: Build
+
+on:
+  pull_request:
```

We can also add types here this way we can always run CI jobs for new commits
```suggestion
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
```

---

sure thanks made the changes